.github/workflows/client.yml

```diff
@@ -67,34 +67,34 @@ jobs:
       - name: Coverage
         run: yarn --cwd client coverage
 
-  publish:
-    name: Publish client
-    needs:
-      - check
-      - test
-    runs-on: ubuntu-latest
-    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
-    steps:
-      - uses: actions/checkout@v2
-      - name: Cache node_modules
-        id: cache-modules
-        uses: actions/cache@v1
-        with:
-          path: node_modules
-          key: 12.x-${{ runner.OS }}-build-${{ hashFiles('package.json') }}
-      - name: Install
-        if: steps.cache-modules.outputs.cache-hit != 'true'
-        run: yarn install
-      - name: Build client
-        run: yarn --cwd client build
-
-      - name: Publish client
-        uses: mikeal/merge-release@master
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
-          SRC_PACKAGE_DIR: client
-          DEPLOY_DIR: client
+  # publish:
+  #   name: Publish client
+  #   needs:
+  #     - check
+  #     - test
+  #   runs-on: ubuntu-latest
+  #   if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - name: Cache node_modules
+  #       id: cache-modules
+  #       uses: actions/cache@v1
+  #       with:
+  #         path: node_modules
+  #         key: 12.x-${{ runner.OS }}-build-${{ hashFiles('package.json') }}
+  #     - name: Install
+  #       if: steps.cache-modules.outputs.cache-hit != 'true'
+  #       run: yarn install
+  #     - name: Build client
+  #       run: yarn --cwd client build
+
+  #     - name: Publish client
+  #       uses: mikeal/merge-release@master
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #         NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+  #         SRC_PACKAGE_DIR: client
+  #         DEPLOY_DIR: client
```

Comment:
Please create an issue to re-instate.

Reply:
https://github.com/ipfs-shipyard/nft.storage/issues/66